[2023-01-05 03:49:49,868] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:49:44.028650+00:00 [queued]>
[2023-01-05 03:49:49,882] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:49:44.028650+00:00 [queued]>
[2023-01-05 03:49:49,882] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 03:49:49,882] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-05 03:49:49,882] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 03:49:49,906] {taskinstance.py:1377} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc_task> on 2023-01-05 03:49:44.028650+00:00
[2023-01-05 03:49:49,916] {standard_task_runner.py:52} INFO - Started process 2539 to run task
[2023-01-05 03:49:49,925] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'btc_data_pipeline_dag', 'create_cluster_dataproc_task', 'manual__2023-01-05T03:49:44.028650+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/btc_data_pipeline_dag.py', '--cfg-path', '/tmp/tmp1k8yb2jd', '--error-file', '/tmp/tmptvmzdkgu']
[2023-01-05 03:49:49,927] {standard_task_runner.py:80} INFO - Job 58: Subtask create_cluster_dataproc_task
[2023-01-05 03:49:50,027] {task_command.py:370} INFO - Running <TaskInstance: btc_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-05T03:49:44.028650+00:00 [running]> on host 94450fd8b415
[2023-01-05 03:49:50,202] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=btc_data_pipeline_dag
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T03:49:44.028650+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-05T03:49:44.028650+00:00
[2023-01-05 03:49:50,204] {dataproc.py:590} INFO - Creating cluster: move-btc-data
[2023-01-05 03:49:50,205] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-05 03:52:38,277] {dataproc.py:534} INFO - Cluster created.
[2023-01-05 03:52:38,309] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=btc_data_pipeline_dag, task_id=create_cluster_dataproc_task, execution_date=20230105T034944, start_date=20230105T034949, end_date=20230105T035238
[2023-01-05 03:52:38,349] {local_task_job.py:156} INFO - Task exited with return code 0
